tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,127 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+import pytest
+import re
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from threading import Thread
+
+import app.app as app
+
+
+@pytest.fixture(scope='module')
+def test_app():
+
+    # Load the environment variables.
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+
+    # Create the new board & update the board id environment variable.
+    response = create_trello_board('test_board')
+    board_id = response['id']
+    organization_id = response['idOrganization']
+    os.environ['BOARD_ID'] = board_id
+
+
+    # Create app instance using newly created board.
+    application  = app.create_app()
+
+    # Start the app in its own thread.
+    thread = Thread(target = lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+
+    # Tear down.
+    thread.join(1)
+    delete_trello_board(board_id)
+    delete_trello_organization(organization_id)
```

Comment:
This seems a little unnecessary and a little risky since you didn't specify an organisation.  For me organization_id is null.